library_samples/Python3/ocs_sample_library_preview/DataViews.py

```diff
@@ -267,58 +286,62 @@ def getDataInterpolated(
         params = {
             "count": count,
             "form": form,
-            "continuationToken": continuationToken,
             "startIndex": startIndex,
             "endIndex": endIndex,
-            "interval": interval,
+            "interval": interval
         }
-        response = requests.get(
-            self.__getDataInterpolated.format(
-                tenant_id=self.__baseClient.tenant,
-                namespace_id=namespace_id,
-                dataView_id=dataView_id,
-            ),
-            headers=self.__baseClient.sdsHeaders(),
-            params=params,
-        )
+        response = {}
+        if url:            
+            response = requests.get(url)
```

Comment:
What about headers=self.__baseClient.sdsHeaders() ? Is nextPage URL sufficient? I don't know the answer but my guess would be you still need headers.

Reply:
yea... i didn't test it... you are right